src/routes/docs/tutorials/react/step-6/+page.markdoc

```diff
@@ -5,7 +5,7 @@ description: Add a database to your React application using Appwrite Web SDK.
 step: 6
 ---
 # Create collection {% #create-collection %}
-In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store our ideas.
+In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store our ideas. To authorize users to view, create, and delete documents, set the collection's permissions for the user's needs.
```

Comment:
Please make the instructions clearer for permissions. Let's turn the part after the screenshot to a sub-section named Attributes. After the Attributes section, add a Permissions section and make the instructions clear. Any user should be able to create documents. Document security should be enabled so that the permissions for each document is different and each user can have access to their own documents.